taskcluster/docker/toolchain-build/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM $DOCKER_IMAGE_PARENT
+LABEL maintainer="Mozilla Release Engineering <release+docker@mozilla.com>"
+
+RUN apt-get update -qq \
+    # We need to install tzdata before all of the other packages. Otherwise it will show an interactive dialog that
+    # we cannot navigate while building the Docker image.
+    && apt-get install -y tzdata \
+    && apt-get install -y wget \
+                          curl \
+                          zip \
+                          build-essential \
+                          gcc \
+                          g++ \
+                          make \
+                          cmake \
+                          libboost-dev \
+                          libboost-all-dev \
+                          zstd \
+                          tar \
+                          libxml2 \
+                          libhunspell-dev \
+    && apt-get clean
+
+RUN locale-gen "$LANG"
+
+RUN pip install zstandard
```

Comment:
@bhearsum @andrenatal I'm curious whether we want to preserve running locally using Snakemake and running on Slurm. If yes, should we add `zstandard` to `envs/base.yml`?

Reply:
That's an excellent question. I think we certainly need to be able to run both for a period of time. The changes to the pipeline scripts are simple enough (thanks to the command line options being the same) that perhaps we can support both there?

Reply:
(An alternative would be to run either Taskcluster or the Slurm pipeline off of a separate branch.)

Reply:
Echoing some discussion from Slack -- it sounds like it's useful in the long term to maintain support for gzip to make it easier for others to run our scripts/pipeline. I'm going to update this to continue to use gz as the default compression mechanism, with a way to override it with zstd.